library/datetime/strftime_spec.rb

```diff
@@ -33,19 +33,9 @@
     @time.strftime("%Z").should == "+00:00"
   end
 
-  # %v is %e-%b-%Y for Date/DateTime
-  version_is date_version, ""..."3.2" do #ruby_version_is ""..."3.1" do
```

Comment:
Is it correct to remove this section guarded with `version_is date_version, ""..."3.2"`?